src/app/app.module.ts

```diff
@@ -14,15 +17,20 @@ import{RouterModule} from '@angular/router';
     DashboardComponent,
     EditComponent,
     StatsComponent
+    AddComponent
   ],
   imports: [
     BrowserModule,
     RouterModule.forRoot([
       {path: '', pathMatch: 'full', redirectTo: 'dashboard'},
       {path:'dashboard',component:DashboardComponent},
       {path:'edit/:id',component:EditComponent},
+      {path:'add', component: AddComponent }
```

Comment:
Add component in this pull ? it was in your previous pull. again ?

Reply:
in previous pull no seprate component made for add so i made the separte file and the their component